src/conditional-flow/string-conditions.js

```diff
@@ -1,41 +1,71 @@
 // This function should return true if the passed string is equal to "Hello"
-function isHello (val1) {
-
+function isHello(val1) {
   // TODO: write code in this function body to pass the tests
-
+  if (val1 === "Hello") {
+    return true;
+  }
+  if (val1 !== "Hello") {
+    return false;
+  }
 }
-
 // This function should return true if the passed string is not equal to "Hello"
-function isNotHello (val1) {
-
+function isNotHello(val1) {
   // TODO: write code in this function body to pass the tests
-
+  if (val1 !== "Hello") {
+    return true;
+  }
+  if (val1 === "Hello") {
+    return false;
+  }
 }
-
 // This function should return true if the string val1 is is longer
 // than string val2
-function isLongerThan (val1, val2) {
-
+function isLongerThan(val1, val2) {
   // TODO: write code in this function body to pass the tests
-
+  if (val1.length > val2.length) {
```

Comment:
You could simplify this function by just returning this condition. You wouldn't need the other if statements then:
```js
return val1.length > val2.length
```